Makefile

```diff
@@ -1096,6 +1096,80 @@ eswikibooks_tuning_reports: \
 		tuning_reports/eswikibooks.damaging.md
 		tuning_reports/eswikibooks.goodfaith.md
 
+############################# Spanish Wikiquote ################################
+
+# From https://quarry.wmflabs.org/query/23421
+datasets/eswikiquote.sampled_revisions.12k_2017.json:
+	wget -qO- https://quarry.wmflabs.org/run/219894/output/0/json-lines?download=true > $@
+
+datasets/eswikiquote.revisions_for_review.5k_2017.json: \
+		datasets/eswikiquote.autolabeled_revisions.12k_2017.json
+	( \
+	 cat $< | \
+	 grep '"needs_review": true' | \
+	 shuf -n 2500; \
+	 cat $< | \
+	 grep '"needs_review": false' | \
+	 shuf -n 2500 \
+	) | shuf > $@
```

Comment:
We should not use this pattern anymore.  It is out-dated.

Reply:
hmm, what is the new strategy? Can you point out to a wiki that uses it?

Reply:
Done, looked at old PRs. Hope that's what you mean.